src/main/java/org/broadinstitute/hellbender/tools/picard/sam/CreateSequenceDictionary.java

```diff
@@ -63,6 +66,11 @@
         if (URI == null) {
             URI = "file:" + REFERENCE_SEQUENCE.getAbsolutePath();
         }
+        if (OUTPUT == null) {
+            // determine the name for the dict file in the same way as CachingIndexedFastaSequenceFile.checkAndCreate
+            final String fastaExt = REFERENCE_SEQUENCE.getAbsolutePath().endsWith("fa") ? "\\.fa$" : "\\.fasta$";
```

Comment:
@magic Can you use FilenameUtils.getBaseName from apache commons-io here instead?